templates/prescription/general_edit.html

```diff
@@ -256,10 +265,22 @@
 {$ENDING_JAVASCRIPT}
 </script>
 </form>
+<script type="text/javascript" src="{$WEBROOT}/public/assets/jquery-min-1-9-1/index.js"></script>
+<script type="text/javascript" src="{$WEBROOT}/library/js/jquery-ui.min.js"></script>
 {literal}
 <!-- for the fancy jQuery stuff -->
 <script type="text/javascript">
 
+$(function() {
+    
+    //Drug autocomplete
+    $(".drug").autocomplete({
```

Comment:
It took a while to figure out how to get the autocomplete to work.  I changed the target to wDrug just to make it easier to identify that the class is being used to invoke the jquery.